go/trivial.go

```diff
@@ -3,33 +3,51 @@ package main
 import (
 	"fmt"
 	"os"
+	"strings"
 	"time"
 
-	"github.com/uber/jaeger-client-go"
+	jaeger "github.com/uber/jaeger-client-go"
 	"github.com/uber/jaeger-client-go/config"
 
-	"github.com/opentracing/opentracing-go"
+	opentracing "github.com/opentracing/opentracing-go"
 )
 
+func errExit(errstr string) {
+	fmt.Println(errstr)
+	os.Exit(1)
+}
+
 func main() {
+	var (
+		hostPort string
+	)
+
+	if len(os.Args) == 2 {
+		hostPort = os.Args[1]
+		if !strings.Contains(os.Args[1], ":") {
+			hostPort += ":6831"
+		}
+	}
+
```

Comment:
too much complexity for an example. Better to use stdlib
```go
import "flag"
func main() {
    var hostport = flag.String("hostport", ":6831", "Jaeger UDP host:port")
    flag.Parse()
```

the homepage would need to change to 
```shell
$ go run ./trivial.go -hostport $DOCKER_IP
```

Reply:
that would still make the example incorrect; fixing the code to meet the example was the initial point of this patch. If someone wants to change the example, sure.